src/map/skill.cpp

```diff
@@ -10459,7 +10486,11 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		clif_skill_nodamage(src,bl,skill_id,skill_lv,
 			sc_start(src,bl,type,100,skill_lv,skill_get_time(skill_id,skill_lv)));
 		if (hd)
+#ifdef RENEWAL
+			skill_blockhomun_start(hd, skill_id, skill_get_cooldown(skill_id,skill_lv));
```

Comment:
```
	case HAMI_BLOODLUST:
	case HFLI_FLEET:
	case HFLI_SPEED:
	case HLIF_CHANGE:
	case MH_ANGRIFFS_MODUS:
	case MH_GOLDENE_FERSE:
```
All have no cooldown defined in skill db.
I think maybe aleos commented on the wrong location.

Reply:
gotcha, corrected these

Reply:
I have reverted all cooldown related changes for the time being and opened #7605.
As this should be revisited outside of the scope of this PR.